src/react-apps/applications/service-development/src/features/deploy/components/deployPaper.tsx

```diff
@@ -0,0 +1,466 @@
+              />
+            </div>
+          </React.Fragment>
+        );
+
+      case inSyncStatus.behind:
+        return (
+          <React.Fragment>
+            <Typography variant='h2' className={classes.listItemTitle}>
+              {getLanguageFromKey('deploy_to_test.changes_made_by_others_in_your_organisation_title', props.language)}
+            </Typography>
+            <Typography variant='body1'>
+              {/* tslint:disable-next-line:max-line-length */}
+              {getLanguageFromKey('deploy_to_test.changes_made_by_others_in_your_organisation_is_not_visible_in_altinn_studio', props.language)}
+            </Typography>
+            <div style={{ marginTop: 20, marginBottom: 20 }}>
+              <VersionControlContainer
+                language={props.language}
+                type='fetchButton'
+              />
+            </div>
+          </React.Fragment>
+        );
+
+      default:
+        return null;
+    }
+  };
+
+  const renderCSharpCompilesText = (param: boolean) => {
+    const altinnWindow: IAltinnWindow = window as IAltinnWindow;
+    const { org, service } = altinnWindow;
+
+    switch (param) {
+
+      case true:
+        return (
+          <Typography variant='h2' className={classes.listItemTitle}>
+            {getLanguageFromKey('deploy_to_test.check_csharp_compiles_true_title', props.language)}
+          </Typography>
+        );
+
+      case false:
+        return (
+          <React.Fragment>
+            <Typography variant='h2' className={classes.listItemTitle}>
+              {getLanguageFromKey('deploy_to_test.check_csharp_compiles_false_title', props.language)}
+            </Typography>
+            <div style={{ margin: '6px 0px 12px 10px' }}>
+              {props.cSharpCompileStatusUniqueFilenames.map((file) => (
+                <Typography variant='body1' key={file}>
+                  {file}
+                </Typography>
+              ))}
+            </div>
+            <Typography variant='body1'>
+              {getLanguageFromKey('deploy_to_test.check_csharp_compiles_false_body_part1', props.language)}&nbsp;
+              <a href={`/${org}/${service}`} className={classes.link} target='_blank'>
+                {/* TODO: Remember to change text when file edit page is available */}
+                {getLanguageFromKey('deploy_to_test.check_csharp_compiles_false_body_part2', props.language)}<AltinnIcon
+                  isActive={true}
+                  iconClass='ai ai-arrowrightup'
+                  iconColor={theme.altinnPalette.primary.black}
+                  iconSize={30}
+                  padding='0px 0px 4px 0px'
+                />
+              </a>
+            </Typography>
+          </React.Fragment>
+        );
+
+      default:
+        return null;
+    }
+  };
+
+  const renderDeploySuccess = (env: string) => {
+    const altinnWindow: IAltinnWindow = window as IAltinnWindow;
+    const { org, service } = altinnWindow;
```

Comment:
Tidier with deconstruction on the same line.
const { org, service } = window as IAltinnWIndow;

Reply:
Fixed.